### PR TITLE
rm wget dependency, update python pkg slightly.

### PR DESCRIPTION
**Pull request by artkiver**
remove wget dependency; use curl since it is a given on macOS.
Update python to 3.9.13 which is the most recent 3.9.x build with a prebuilt macosx pkg on https://www.python.org/ftp/python/

TODO: update to 3.9.18? 3.12? 3.13?

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ set -e
 
 cd "$(dirname "$0")"
 
-PYTHON_VER=3.9.6
+PYTHON_VER=3.9.13
```
Comment (eiln) [inline build.sh:8]: Separate commits for unrelated changes. Python version bump should not be bundled with a shell script change, if at all.

File: build.sh [hunk 2]
```diff
@@ -43,7 +43,7 @@ echo "Downloading installer components..."
 
 cd "$DL"
 
-wget -Nc "$PYTHON_URI"
```
Comment (eiln) [inline build.sh:46]: `wget -Nc $f` != `curl $f`. I would add `-C -` for `-c` (continue). For `-N` (timestamping), it seems like there's a way to wrap it with `-z` and `-r`, not sure if it's pretty. https://stackoverflow.com/a/57173943

**Comment by eiln** [comment]
Need DCO. Use `git commit -s`

**Comment by artkiver** [comment]
> Need DCO. Use `git commit -s`

No thanks.

Also see:

https://www.jwz.org/blog/2024/03/enumerating-objects-counting-objects-compressing-objects-receiving-objects-resolving-deltas-updating-files-fast-forward/#comment-248132

Anyway, given that @marcan thinks I am being "insufferably confrontational" I guess feel free to ignore input I have to offer and write it off as hateful nonsense instead of heartfelt criticism from someone with more experience in this field than Hector has alive on this planet.

**Comment by marcan** [comment]
Backstory, for those who are wondering, is that this person started complaining about everything about the installer on Fedi from `curl|sh` to the need for the sudo password, and apparently they think they know better than the entire Asahi development team.

Given the refusal to sign the DCO, and the essentially no-op changes made, there is no value left in this PR.
